src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -1369,6 +1369,14 @@ public ErrorDiagnostic UnknownModuleReferenceScheme(string badScheme, ImmutableA
                 TextSpan,
                 "BCP231",
                 "Using resource-typed parameters and outputs requires enabling EXPERIMENTAL feature BICEP_RESOURCE_TYPED_PARAMS_AND_OUTPUTS_EXPERIMENTAL.");
+
+            public FixableErrorDiagnostic SymbolicNameShadowsAKnownFunction(string name, string knownFunctionNamespace, string knownFunctionName) => new(
+                TextSpan,
+                "BCP232",
+                $"The name \"{name}\" is not a function. Did you mean {knownFunctionNamespace}.{knownFunctionName}()",
```

Comment:
Please could you use quotation marks and a question mark?
```suggestion
                $"The name \"{name}\" is not a function. Did you mean \"{knownFunctionNamespace}.{knownFunctionName}()\"?",
```